### ISSUE TITLE
Unknown column 'modResource.alias_visible' in 'field list'

### ISSUE THREAD

**Issue by Ibochkarev** [modxcms/revolution#15141]
### Summary
After a gradual update of СMS from version:
2.5.5 -> 2.6.0 -> 2.6.5 -> 2.7.0 -> 2.7.3 an error appeared in the journal and the admin panel stopped working. White screen.

### Observed behavior
Update 2.5.5 -> 2.6.0 -> 2.6.5 -> 2.7.0 -> 2.7.3

``` php
[2020-06-22 13:59:47] (ERROR @ /home/egg00/***/www/core/xpdo/om/xpdoobject.class.php : 240) Error 42S22 executing statement: 
Array
(
    [0] => 42S22
    [1] => 1054
    [2] => Unknown column 'modResource.alias_visible' in 'field list'
)

[2020-06-22 13:59:47] (ERROR @ /home/egg00/***/www/core/xpdo/om/xpdoobject.class.php : 240) Error 42S22 executing statement: 
Array
(
    [0] => 42S22
    [1] => 1054
    [2] => Unknown column 'modDocument.alias_visible' in 'field list'
)
```

### Expected behavior
I took the advice and fulfilled the SQL request and it worked.

```sql
ALTER TABLE `modx_site_content` ADD `alias_visible` TINYINT( 1 ) UNSIGNED NOT NULL DEFAULT '1' AFTER `alias` ;
```

https://github.com/modxcms/revolution/issues/14219#issuecomment-450200690

### Related issue(s)/PR(s)
#15035 

### Environment

MODX 2.7.3, apache, mysql 5.7.16-10, php 7.2.31

**Comment by JoshuaLuckers** [comment]
The column should have been added by installing 2.7.0. 

Related #14382

**Comment by Ibochkarev** [comment]
> The column should have been added by installing 2.7.0.

I agree, but this did not happen

**Comment by JoshuaLuckers** [comment]
If you have time to test: does it also happen if you upgrade from 2.6.5 immediately to 2.7.3?

**Comment by Ibochkarev** [comment]
> If you have time to test: does it also happen if you upgrade from 2.6.5 immediately to 2.7.3?

Ok i will check

**Comment by Ibochkarev** [comment]
@JoshuaLuckers I checked. All OK. It’s strange. Perhaps something is wrong with the hosting.

```
The configuration file was successfully written.

Permissions on the configuration file were successfully updated.

Cleanup of obsolete files completed.
Deleted 1 file (s) and 0 directory (s).

Table for class "modResource" updated successfully
A new column `alias_visible` has been added to the table` `8qjURYjzSGVl_site_content``.

Table for class "modResource" updated successfully
Changed the column `description` in the table` `8qjURYjzSGVl_site_content``.

Table for class "modUser" updated successfully
The column `password` in the table` `8qjURYjzSGVl_users`` has been changed.

Table for class "modUser" updated successfully
The column `cachepwd` in the table` `8qjURYjzSGVl_users`` has been changed.

Table for class "modUser" updated successfully
The column `hash_class` in the table `8qjURYjzSGVl_users` has been changed.
```

**Comment by Ibochkarev** [comment]
Updated two more old sites. No errors found

**Comment by Jako** [comment]
Maybe this occured because of the bug solved in #14396

**Comment by rthrash** [comment]
This issue has been mentioned on **MODX Community**. There might be relevant details there:

https://community.modx.com/t/manager-error-resource-id-not-found/4750/2
